### ISSUE TITLE
sidebar is still white even whit dark colorschemes

### ISSUE THREAD

**Issue by Toni500github** [vednoc/dark-gitlab#2]
![image](https://github.com/vednoc/dark-gitlab/assets/88878648/46e78814-6612-42ed-a3aa-0f007a8a6a49)
please fix, I really like this repo

**Comment by JuniorLoch** [comment]
Same here.

**Comment by perlun** [comment]
Yep, the theme regretfully doesn't seem to work with the "new GitLab UI" that was introduced a while ago. :cry:
